skrifa/src/outline/glyf/mod.rs

```diff
@@ -54,11 +54,16 @@ pub struct Outlines<'a> {
     units_per_em: u16,
     os2_vmetrics: [i16; 2],
     prefer_interpreter: bool,
+    pub(crate) prevent_fractional_scaling: bool,
 }
 
 impl<'a> Outlines<'a> {
     pub fn new(font: &FontRef<'a>) -> Option<Self> {
-        let loca = font.loca(None).ok()?;
+        let head = font.head().ok()?;
+        // If bit 3 of head.flags is set, then we round ppems when
+        // scaling
+        let prevent_fractional_scaling = head.flags() & 0x8 != 0;
```

Comment:
I was expecting to see use of an enum for the flags. https://docs.rs/read-fonts/latest/read_fonts/tables/head/type.Head.html#method.flags helpfully advises me to use "the flags enum" but doesn't link to it. Bleh.

Reply:
Happy to "enumize" head.flags in a follow up